src/win/core.c

```diff
@@ -113,8 +113,17 @@ int uv_loop_init(uv_loop_t* loop) {
   /* Initialize libuv itself first */
   uv__once_init();
 
+  /* determine NumberOfConcurrentThreads for I/O completion port, 2 works better for single core */
+  int coreCount = -1, ioCompletionThreads = 1;
+  char *coreCountStr = getenv("NUMBER_OF_PROCESSORS");
```

Comment:
AFAICT we prefer `uv_os_getenv`, but any of these introduce thread unsafety... I'd want to know what others think.